### ISSUE TITLE
String format arity checker wrong w/ lvars

### ISSUE THREAD

**Issue by zenspider** [bbatsov/rubocop#2319]
Rubocop shouldn't assume that the arity of the string is 1 in the case of a string being formatted against an lvar. Either it should relax or it should actually trace lvars and check the arity there. Example from minitest:

```ruby
        data = [mu_pp(act), act.object_id, mu_pp(exp), exp.object_id]
        "Expected %s (oid=%d) to be the same as %s (oid=%d)" % data
```

Note, this is only done this way to respect line length.

**Comment by bbatsov** [comment]
I think this was fixed on `master`.

**Comment by bbatsov** [comment]
My bad - actually it was only reported https://github.com/bbatsov/rubocop/issues/2307. I'll close this ticket as a duplicate.
